pkgs/tools/system/netdata/mosquitto.nix

```diff
@@ -0,0 +1,38 @@
+# The Netdata project uses a custom fork of Mosquitto that
+# incorporates support for some callback functionality they
+# require.
+#
+# Also see https://github.com/netdata/netdata/issues/8961
+
+{ stdenv, fetchFromGitHub, c-ares, cmake, libwebsockets_3_2, openssl }:
+
+stdenv.mkDerivation rec {
+  pname = "mosquitto";
+  version = "1.6.8_Netdata-5";
+
+  src = fetchFromGitHub {
+    owner = "netdata";
+    repo = "mosquitto";
+    rev = "v.${version}";
+    sha256 = "0ayxnf0wa1nq52ana7san62zzkcm08ss8pmpx8hzp9yi0zl5mpal";
+  };
+
+  buildInputs = [ c-ares libwebsockets_3_2 openssl ];
+  nativeBuildInputs = [ cmake ];
+
+  cmakeFlags = [
+    "-DWITH_THREADING=YES"
+    "-DWITH_WEBSOCKETS=YES"
+    "-DWITH_STATIC_LIBRARIES=YES"
+    "-DWITH_DOCS=NO"
+  ];
+
+  makeFlags = [
+    "-Clib"
+  ];
+
+  postInstall = ''
+    ln $out/lib/libmosquitto_static.a $out/lib/libmosquitto.a
+    ln $out/include/mosquitto.h $out/lib/mosquitto.h
+  '';
```

Comment:
Not sure if we want to add a meta section here.

Reply:
Sorry for keeping you waiting, I don't think I got any notifications for your comments. Honestly, I think it should be fine not having the meta section, especially since I would like to remove this package from Nixpkgs as soon as [ACLK-NG](https://github.com/NixOS/nixpkgs/pull/92782#discussion_r583834062) is present in the Netdata package provided by Nixpkgs.

Reply:
Packages require a meta section.

Reply:
I mean, this package will likely disappear in a few weeks, plus it's only used by netdata.